ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/CVEsTable.tsx

```diff
@@ -203,16 +237,39 @@ function CVEsTable({
                                     />
                                 </Td>
                                 <Td dataLabel="Affected images">
-                                    {/* TODO: fix upon PM feedback */}
                                     {affectedImageCount}/{unfilteredImageCount} affected images
                                 </Td>
                                 <Td dataLabel="First discovered">
                                     <DateDistanceTd date={firstDiscoveredInSystem} />
                                 </Td>
+                                {showExceptionMenuItems && (
+                                    <Td className="pf-u-px-0">
+                                        <ActionsColumn
+                                            items={[
+                                                {
+                                                    title: 'Defer CVE',
+                                                    onClick: () =>
+                                                        cveTableActionHandler({
+                                                            type: 'DEFERRAL',
+                                                            cves: [cve],
+                                                        }),
+                                                },
+                                                {
+                                                    title: 'Mark false positive',
```

Comment:
Similar question as above about **Mark CVE as false positive** notice absence of **as** here.

Reply:
I'll bring this up with UX and see if we can get consistency with the wording here. I didn't notice the subtle differences at first across the options, but it might just be minor inconsistencies in the mocks.